src/wazuh_db/schema_upgrade_v9.sql

```diff
@@ -0,0 +1,20 @@
+/*
+ * SQL Schema for upgrading databases
+ * Copyright (C) 2015, Wazuh Inc.
+ *
+ * Jun 27, 2022
+ *
+ * This program is a free software, you can redistribute it
+ * and/or modify it under the terms of GPLv2.
+*/
+
+DROP TABLE IF EXISTS sys_hotfixes;
```

Comment:
I also thought about distinct to avoid the checksum constraint warning, but I faced the situation that the table we want to upgrade may not have the checksum. Should we create a query smart enough to verify first if the table has that column?

Reply:
The problem here is that the hotfixes table does not duplicate by null, It does it by the "scan_time" timestamp, if we remove this from pks, the new insertion can give constraint fail.

imagine the case

|scan_id|hotfix|scan_time|
| ------------- | ------------- | ------------- |
|0|KB12345678|2017/06/06 22:22:22|
|0|KB12345678|2018/07/07 23:23:23|


If we do a select distinct we continue with different data.

I recommend doing a clean of the table if it has duplicates, we can check this with having count and the old PKs (scan_id, hotfix, scan_time)

Reply:
Considering a scenario where we have duplicate checksum values. 
![2](https://user-images.githubusercontent.com/13010397/177200822-9f2ce2b7-ea50-4978-a8a1-b7dd2ea08e6e.png)
After the upgrade, it will add the older value if it has more than one. 
![3](https://user-images.githubusercontent.com/13010397/177200826-3ec9fa8e-218a-4f0c-aa5c-5da60b87d83d.png)